kubernetes/README.md

```diff
@@ -0,0 +1,71 @@
+# HabitaRPG in Kubernetes
```

Comment:
Nit: this whole file should consistently capitalize words such as Kubernetes, AWS, Habitica, etc.